packages/react-components/src/styles/index.ts

```diff
@@ -245,6 +255,22 @@ export default createGlobalStyle<Props & ThemeProps>(({ theme, uiHighlight }: Pr
     }
   }
 
+  .ui--SummaryBox {
+    > section {
+      background: ${hexToRGB(getHighlight(uiHighlight), '0.08')};
```

Comment:
Why the background?

So at this point it actually catches your eye, why the summary is meant to be background info, not dominate.

Reply:
I've removed the summary background ✅